src/components/AccountDrawer/AccountDrawerDisplay.test.tsx

```diff
@@ -30,255 +30,155 @@ import {
-    // Remove all assets in the store
-    store.dispatch(assetsSlice.actions.reset());
-
-    render(<AccountCard account={selectedAccount} />);
+    it("is displayed", async () => {
+      render(<AccountCard account={selectedAccount} />);
+      await waitFor(() => expect(screen.getByTestId("asset-panel-tablist")).toBeVisible());
 
-    await act(() => user.click(screen.getByTestId("account-card-tokens-tab")));
-    expect(screen.getByTestId("asset-panel")).toBeInTheDocument();
+      const tzktLink = within(screen.getByTestId("asset-panel-tablist")).getByRole("link", {
+        name: "View on Tzkt",
+      });
+      expect(tzktLink).toBeVisible();
+    });
 
-    const { getByText } = within(screen.getByTestId("asset-panel"));
-    expect(getByText("No tokens to show")).toBeInTheDocument();
-  });
+    it("opens correct page according to selected network", async () => {
+      render(<AccountCard account={selectedAccount} />);
+      await waitFor(() => expect(screen.getByTestId("asset-panel-tablist")).toBeVisible());
 
-  test("tokens tab should display token infos correctly", async () => {
-    render(<AccountCard account={selectedAccount} />);
-    const tokenTiles = screen.getAllByTestId("token-tile");
-    await waitFor(() => {
-      expect(tokenTiles).toHaveLength(4);
+      const tzktLink = within(screen.getByTestId("asset-panel-tablist")).getByRole("link", {
+        name: "View on Tzkt",
+      });
+      const expectedLink = "https://ghostnet.tzkt.io/" + selectedAccount.address.pkh;
+      expect(tzktLink).toHaveProperty("href", expectedLink);
     });
 
-    {
-      const { getByTestId } = within(tokenTiles[0]);
-      expect(getByTestId("token-name")).toHaveTextContent("Hedgehoge");
-      expect(getByTestId("token-balance")).toHaveTextContent("10,000.000000");
-    }
-
-    {
-      const { getByTestId } = within(tokenTiles[1]);
-      expect(getByTestId("token-name")).toHaveTextContent("tzBTC");
-      expect(getByTestId("token-balance")).toHaveTextContent("0.00002205");
-    }
-
-    {
-      const { getByTestId } = within(tokenTiles[2]);
-      expect(getByTestId("token-name")).toHaveTextContent("FA1.2 token");
-      expect(getByTestId("token-balance")).toHaveTextContent("123");
-    }
-
-    {
-      const { getByTestId } = within(tokenTiles[3]);
-      expect(getByTestId("token-name")).toHaveTextContent("youves uUSD");
-      expect(getByTestId("token-balance")).toHaveTextContent("0.01921875");
-    }
-  });
+    it("is updated on network change", async () => {
+      render(<AccountCard account={selectedAccount} />);
+      await waitFor(() => expect(screen.getByTestId("asset-panel-tablist")).toBeVisible());
 
-  it("should display nfts under nfts tab", async () => {
-    const user = userEvent.setup();
-    render(<AccountCard account={selectedAccount} />);
+      {
+        await act(() => store.dispatch(networksActions.setCurrent(MAINNET)));
+        expect(screen.getByTestId("asset-panel-tablist")).toBeVisible();
```

Comment:
since you are querying the same testId on the next line it's a good idea to store it in a variable. easier to read, refactor, and it's faster

Reply:
I don't think we really need line 101 - there is no waitFor & if something goes wrong with "asset-panel-tablist", next line will fail.